core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -10,9 +10,12 @@
     </div>
     <div class="topic-details" ng-style="{'background-color': $ctrl.getDarkerThumbnailBgColor()}" ng-class="{'mask-effect': !$ctrl.isPublished()}">
       <span class="topic-name"> <[$ctrl.getTopicSummary().getName()]> </span>
-      <span translate="I18N_TOPIC_VIEWER_CHAPTERS"
```

Comment:
Hm do we use `I18N_TOPIC_VIEWER_CHAPTERS` anywhere? If not maybe we can drop this key from the .json files.

Reply:
In my previous PR, I made a change to a separate file `story-summary-tile` so that it uses `I18N_TOPIC_VIEWER_CHAPTERS`. (originally it used `I18N_TOPIC_VIEWER_CHAPTER + "s"`)